### DISCUSSION TITLE
SSD1306 OLED display & Arduino Due

### DISCUSSION THREAD

**Discussion by rtookl** [olikraus/u8g2#2082] [General]
Hi Oliver,

I am a bit new to this whole thing. I am trying to use a SSD1306 display with Arduino Due board. with Arduino Uno it works fine and but I guess I am not using the right constructor for Due. Can you help me?

Regards,
R

**Comment by rtookl** [comment]
Problem solved. For I2C com, pins SDA and SCL need to be used

**Reply by olikraus** [reply]
Yes, probably the pin numbers do differ between Due and Uno...
